docs/guide/release.md

```diff
@@ -0,0 +1,51 @@
+# Release guidelines
+
+
+odfe-cli versions are expressed as **x.y.z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version, following [Semantic Versioning](https://semver.org/) terminology. 
+
+**The major, minor & patch version is related to odfe-cli compatibility, instead of odfe’s compatibility.**
+
+ Note: We only backport applicable fixes, including security fixes to released branches, depending on severity and feasibility, everything else needs to go in next available minor release train.
+
+AI: Action Item for change by plugin owners
+
+**Q: What goes into new patch version release train?**
+
+* Known bugs that prevent users from executing tasks. 
+    * AI: Fix the bug, write/update unit test and integration test (if desired).
+    * eg: Failed to delete non-running detectors by delete command, failed to handle error.
+* Security vulnerabilities. 
+    * AI: Fix code and merge.
+    * eg: leaking user credentials, insecure interaction with components.
+* cli’s dependency is updated with new patch version which fixes bugs and security issues that affects your feature. ( Treat your dependent library as your library)
+    * AI: Updating your dependency in go.mod file to latest patch version. 
+    * eg: cobra released a new version which fixes some of security vulnerabilities like [#1259](https://github.com/spf13/cobra/pull/1259)
+
+The plugin owner, who merged commit to master, **related to above use case**, should notify admin to initiate new patch release. There is no SLA for patch releases since this is triggered by individual plugin owners.
+
+
+
+**Q: What goes into new minor version release train?**
+
+* Any new commands ( could be new plugin, new sub command from any plugin ). A new command could be new API that is release long ago but being included in new odfe-cli release or new API that will be  released in next odfe release.
+    * AI: Merge PR, update documentation to reflect new changes.
+    * eg: on-board [k-nn](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/20/) plugin , add auto-complete feature for cli commands.
```

Comment:
no space after plugin. It should be  "plugin, "

Reply:
Ack